tests/ParserTest.php

```diff
@@ -74,7 +74,10 @@ public function files()
                     // or a future test of a as-of-now missing feature
                     continue;
                 }
-                $oParser = new Parser(file_get_contents($sDirectory . '/' . $sFileName));
+                $oParser = new Parser(
+                    \file_get_contents($sDirectory . '/' . $sFileName),
+                    Settings::create()->withLenientParsing($sFileName[0] !== '=')
```

Comment:
Instead of having separate sets of fixtures with "magic" prefixes, I'd prefer to communicate this more clearly to the reader, e.g., with a separate testcase that uses the CSS in the test code instead of reading from files. (After all, one of the purposes of tests is to communicate the intended behavior to the reader).

Reply:
I've moved the new tests to the specific test class responsible.  When working within an IDE, having the test content as separate files is no probem, and helps avoid test class files with literally millions of lines to scroll through, which, even if it doesn't cause the computer to break down, is a bit of a PITA.